.github/workflows/ci-unit.yaml

```diff
@@ -46,17 +46,34 @@ jobs:
         with:
           maven-version: 3.6.1
 
+      - name: run unit tests install by skip tests
+        run: mvn clean install -DskipTests
+
+      - name: run unit tests pulsar proxy
+        run: mvn install -DfailIfNoTests=false '-Dtest=!ProxyTest,!ProxyLookupThrottlingTest' -pl pulsar-proxy
```

Comment:
no need to install not just run mvn test for all subsequent actions

Reply:
@aahmed-se PTAL